CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+## Contribution guidelines
+
+Contributions to `becquerel` are welcome and encouraged, whether it is
+reporting bugs, requesting features, or contributing code.
+Please follow these guidelines when contributing to this project.
+
+### Reporting issues
+
+When reporting issues with `becquerel`, please provide a minimum working example to help identify the problem and tag the issue as a `bug`.
+
+### Feature requests
+
+For a feature request, please create an issue and label it as a `new feature`.
+
+### Checklist for code contributions:
```

Comment:
Maybe we should add ` [  ] Spellcheck your code and docu-strings`. I know at least one person not doing so :).

Reply:
Good point. Maybe also requiring `pydocstyle` to check the docstrings.